.github/ISSUE_TEMPLATE/01_question.yml

```diff
@@ -0,0 +1,37 @@
+name: Question/Query
+description: Use this template to file a question/query for the API Spec team that are not bugs or feature requests.
+title: "[Question]"
+
+labels:
+  - question
+
+body:
+  - type: input
+    id: api_spec
+    attributes:
+      label: API Spec and version
+      description: "Path to the API Spec and version"
+      placeholder: "e.g. appconfiguration/data-plane 2023-11-01, or liftrqumulo/Qumulo.Management 1.0.0"
```

Comment:
The placeholder has spaces in it before date. How about we show here full, well-formed file path pointing to relevant `readme.md` ?

There is also aspect of manually-created OpenAPI specs vs TypeSpec specs. In case of TypeSpec, they should point us to the .tsp sources.

Reply:
Could you please provide me with an example of each type of API Spec? I don't have much experience with the various methods of creating API Specs and I want to ensure that the people who complete these templates are providing information that enables you and others who work in this repository to identify the specific spec they are referencing.

Reply:
@ronniegeraghty short answer:
- [Example PR adding new TypeSpec spec](https://github.com/Azure/azure-rest-api-specs/pull/27563/files) and [the dir view](https://github.com/GrahamMThomas/azure-rest-api-specs/tree/eea93183b67d90defa76c8d5eda6c427f957c889/specification/healthdataaiservices/HealthDataAIServices.Deidentification). See also [typespec.io](https://typespec.io/).
- [Example "manual" (no TypeSpec) OpenAPI spec config file: agrifood/resource-manager/readme.md](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/agrifood/resource-manager/readme.md) with the actual OpenAPI spec for **resource-manager** and API version **2021-09-01-preview** [x](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2021-09-01-preview/agfood.json)

Recall that TypeSpec specs generate OpenAPI specs. Similar to how TypeScript sources generate JavasScript sources.

We have at least the following docs about the specs layout, pending review & deduplication:

- https://eng.ms/docs/products/azure-developer-experience/design/api-specs/api-dir-layout
- https://github.com/Azure/azure-rest-api-specs#directory-structure
- https://azure.github.io/azure-sdk/policies_repostructure.html
- TypeSpec-specific: https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/typespec-structure-guidelines.md

Overall, the OpenAPI specs are a complex construct, and depending on the nature of the question, the user may want to point us to various parts.

They can ask about specific OpenAPI spec `.json` in specific version. Version can be a GA (generally available) or a preview. The spec may be resource-manager (ARM) or data-plane. And the spec may be written manually or generated from TypeSpec. The specs are organized by resource provider, and the file tying together the OpenApi specs for given service provider is the readme.md (it is actually a config file read by automation, in spite of the misleading name).

Reply:
@konrad-jamrozik,
Just pinging again on my question above.

Reply:
Discussed elsewhere.

Reply:
@konrad-jamrozik, 
Updated the Spec and Version inputs with recommendations from the discussion channel you shared with me.